packages/pds/tests/_util.ts

```diff
@@ -188,7 +188,7 @@ const basicAuth = (username: string, password: string) => {
 
 // Swap out identifiers and dates with stable
 // values for the purpose of snapshot testing
-export const forSnapshot = (obj: unknown) => {
+export const forSnapshot = (obj: unknown, collapseImgUris?: boolean) => {
```

Comment:
I see this new param mirroring the change in bsky— think should we remove it or implement it here?

Reply:
ah we can remove in both places